query-service-api/src/main/java/org/hypertrace/core/query/service/util/QueryRequestUtil.java

```diff
@@ -129,23 +130,17 @@ public static Filter createTimeFilter(String columnName, Operator op, long value
             Expression.newBuilder()
                 .setLiteral(
                     LiteralConstant.newBuilder()
-                        .setValue(
-                            org.hypertrace.core.query.service.api.Value.newBuilder()
-                                .setString("1:MILLISECONDS:EPOCH"))))
+                        .setValue(Value.newBuilder().setString("1:MILLISECONDS:EPOCH"))))
```

Comment:
All nice and good until you realize this is used in gateway-service.

Reply:
Yeah, I completely removed it in my original change and then found that out the fun way when testing locally. Hence the change to deprecation here.